### PR TITLE
fix(headless): when a tab is selected, the pagination resets to 1

### PR DESCRIPTION
**Pull request by drosenfeld-coveo**
[SFINT-4860](https://coveord.atlassian.net/browse/SFINT-4860)

Small change so when we change tabs, it will reset to the first page

Steps to test:
1. Open a atomic(or quantic) insight panel with multiple tabs and pages
2. go to page 2 on the first tab
3. switch to another tab
4. Page should be changed to 1 after swaping tabs

[SFINT-4860]: https://coveord.atlassian.net/browse/SFINT-4860?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by ThibodeauJF — APPROVED** [review]
Looks good, I don't think the package-lock.json should be part of the PR tho!

you can run something like `git checkout master package-lock.json`

**Review by btaillon — CHANGES_REQUESTED** [review]
Requested change because of my comment.

**Review by btaillon — APPROVED** [review]

File: packages/headless/src/features/pagination/pagination-slice.test.ts [hunk 1]
```diff
@@ -100,6 +101,13 @@ describe('pagination slice', () => {
     expect(finalState.firstResult).toBe(0);
   });
 
+  it('#updateActiveTab sets sets the #firstResult to 0', () => {
```
Comment (jelmedini) [inline packages/headless/src/features/pagination/pagination-slice.test.ts:104]: not sure to understand this sentence, maybe a more explicit description ?

**Review by jelmedini — APPROVED** [review]
looks good !!

File: packages/headless/src/integration-tests/field-suggestions.test.ts [hunk 2]
```diff
@@ -198,7 +198,7 @@ describe('field suggestions', () => {
     });
 
     it('can update captions', async () => {
-      const rawValue = 'pdf';
+      const rawValue = 'PDF';
```
Comment (louis-bompart) [inline packages/headless/src/integration-tests/field-suggestions.test.ts:201]: Done in #2771

**Review by louis-bompart — APPROVED** [review]

**Review by mmitiche — APPROVED** [review]

**Pull request merged**